stages/org.osbuild.fstab

```diff
@@ -14,12 +14,40 @@ This stage replaces /etc/fstab, removing any existing entries.
 import sys
 
 import osbuild.api
+from osbuild.util import ostree
 
 
 SCHEMA = """
 "additionalProperties": false,
 "required": ["filesystems"],
 "properties": {
+  "ostree": {
```

Comment:
The intention is to potentially drop this before declaring it stable, right? Should we add some warning to the docs about that? Or would it be even easier to introduce an org.osbuild.ostree.fstab stage instead that could be marked "tech preview" and easily dropped?

Reply:
My plan was to put this in the release notes, but having it in the stage itself is probably a good idea as well.